blueflood-integration-tests/src/integration-test/java/com/rackspacecloud/blueflood/core/cache/MetadataCacheBatchedModeIntegrationTest.java

```diff
@@ -14,21 +14,21 @@
  *    limitations under the License.
  */
 
-package com.rackspacecloud.blueflood.cache;
+package com.rackspacecloud.blueflood.core.cache;
```

Comment:
Why move to a different package? Tests should be in the same package as the source they're testing.

Reply:
done